database/migrations/create_general-settings_table.php.stub

```diff
@@ -12,6 +12,8 @@ return new class extends Migration
             $table->id();
             $table->string('site_name')->nullable();
             $table->text('site_description')->nullable();
+            $table->string('site_logo')->nullable();
```

Comment:
You should create a new migration because you can't force the user to recreate.

And need to put in the documentation something to force run the command to create new fields with new migration.

And is better to avoid bug to people that have now, create a feature/flag to show these 2 fields only if enable it

Reply:
My bad, I hope my new commit fixes that. Thanks for your feedback.

Reply:
Thanks man, I will see this soon @KaramNassar